generate.py

```diff
@@ -760,15 +760,14 @@ def callback(x, *, done_generating=False):
                     sequential_prefill=generator_args.sequential_prefill,
                     start_pos=start_pos,
                     max_seq_length=max_seq_length,
-                ):
-                    if metrics:
-                        aggregate_metrics["accept_counts"].append(
-                            metrics["accept_counts"]
-                        )
-                    start_pos += y.size(0)
-                    num_tokens_generated += y.size(0)
+                )
+                for y, metrics in gen:
```

Comment:
What is y? can we rename it?

Reply:
This was the original name and I think it works well - x is the input tensor/prompt, y is the output tensor. We could rename it to something else, but not sure if it's in the scope of fixing the FTT.